DatabaseWorkshop.md

```diff
@@ -0,0 +1,185 @@
+# Adding persistance to ToDoBackend with Swift-Kuery-ORM
+
+<p align="center">
+<img src="https://www.ibm.com/cloud-computing/bluemix/sites/default/files/assets/page/catalog-swift.svg" width="120" alt="Kitura Bird">
+</p>
+
+<p align="center">
+<a href= "http://swift-at-ibm-slack.mybluemix.net/"> 
+    <img src="http://swift-at-ibm-slack.mybluemix.net/badge.svg"  alt="Slack"> 
+</a>
+</p>
+
+## Prerequisite
+This is a follow on tutorial to our [ToDoBackend tutorial](https://github.com/IBM/ToDoBackend). Please complete that before proceeding with this tutorial.
+
+## Installing PostgreSQL
+
+Before we start, ensure you have PostgreSQL installed and running. In Terminal, run:
+```
+brew services start postgresql
+```
+
+If it's already running then skip ahead to step 1. If you do not have it installed then run the following from Terminal:
+```
+brew install postgresql
+brew services start postgresql
+```
+
+## Persistance over local storage
+Adding a database connection means your state remains after you close your server down, and depending on where your database is stored, different applications on different machines can access the data (for instance, in a Cloud application).
+
+### Step 1 - Create a Database
+Create a new database for the project called `tododb` using your Terminal:
+```
+createdb tododb
+```
+
+### Step 2 - Updating Package.swift
+Your `Package.swift` needs two new Packages for the PostgreSQL and ORM to work. 
+1. Open the `ToDoServer` > `Package.swift` file
+2. Add the following two lines to the end of the Package.swift's dependencies section:
```

Comment:
Please can you reword to
"Add the following two lines to the end of the dependencies section of the `Package.swift` file:"

Reply:
✅